modules/dashboards/spec/features/project_status_spec.rb

```diff
@@ -118,7 +118,7 @@ def add_project_status_widget
         field.set_to('AT RISK')
 
         # The edit field is toggled and the value saved.
-        expect(page).to have_content('AT RISK')
+        expect(page).to have_content('AT RISK', wait: 5)
```

Comment:
This is not a request for change, but something to think about amongst all of us. It would be nice to agree on a non-arbitrary long wait time for content that is AJAX-dependent and use a constant throughout the app like:

```ruby
AJAX_DEPENDENT_WAIT = 15.seconds # or whatever number we decide to try
expect(page).to have_content('AT RISK', wait: AJAX_DEPENDENT_WAIT)
```

Reply:
Yeah, this is definitely something to think about. Right now we have a lot of arbitrary waiting times, depending on certain test cases.